.circleci/config.yml

```diff
@@ -1010,11 +1034,79 @@ jobs:
             ./Tests/scripts/slack_notifier.sh 'sdk_faild_steps' ./env_results.json
           when: always
 
+  Demisto SDK Nightly - Run against Cortex XSOAR:
+    <<: *container_config
+    resource_class: medium
+    <<: *environment
+    steps:
+      - checkout
+      - setup_remote_docker
+      - *restore_cache
+      - *prepare_environment
+      - *download_configuration
+      - *create_instance
+      - *setup_instance
+      - run:
+          name: Wait until server ready
+          when: always
+          command: |
+            python3 ./Tests/scripts/wait_until_server_ready.py "Demisto Marketplace"
+      - run:
+          name: Set Environment Variables
+          command: |
+            echo "export DEMISTO_API_KEY=$(cat $SECRET_CONF_PATH | jq -r '.temp_apikey')" >> $BASH_ENV
+            echo "export DEMISTO_BASE_URL=\"https://$(cat env_results.json | jq -r '.[0].InstanceDNS')\"" >> $BASH_ENV
+            echo "Server URL: $DEMISTO_BASE_URL"
+      - run:
+          name: Install SDK Master
+          command: pip3 install git+https://github.com/demisto/demisto-sdk.git
+      - run:
+          name: Unlock HelloWorld integration and playbook
+          command: python3 Tests/sdknightly/unlock_entity.py integration HelloWorld
+      - run:
+          name: Create and upload incident type, incident field and a layout to Cortex XSOAR
+          command: python3 Tests/sdknightly/create_entities_for_nightly_sdk.py HelloWorld  # TODO: remove the upload from the script when layout fixed.
+      - *upload_entities_to_cortex_xsoar
+      - run:
+          name: Download entities from Cortex XSOAR
+          command: |
+            demisto-sdk download -o Packs/HelloWorld/ -i HelloWorld --insecure -f
+            demisto-sdk download -o Packs/HelloWorld/ -i HelloWorld-Test --insecure -f
+            demisto-sdk download -o Packs/HelloWorld/ -i "Hello World Incident Test" --insecure -f
+            demisto-sdk download -o Packs/HelloWorld/ -i "Hello World Alert Test" --insecure -f
+      - run:
+          name: Format the entities
+          command: |
```

Comment:
Add a `shell: /bin/bash` line to the command and then you can remove the try and catch.

When specifying `shell: /bin/bash` it will return the last command exit code and will not fail at first failure like it behaves without it.

The command should look like this:
```
- run:
          name: Format the entities
          shell: /bin/bash
          command: |
             demisto-sdk format -i Packs/HelloWorld/ -nv -ud -y
             echo "format exit status: $?" 
```

Reply:
What if the format fails for the right reasons?
Shouldn't you add some logic about that to see that you fail the build in such cases?

Reply:
there's an open bug. after the bug will be fixed we can remove it.

Reply:
Issue https://github.com/demisto/etc/issues/29575